client.go

```diff
@@ -195,6 +195,30 @@ func (c *Client) LsSync(ctx context.Context, opts ...LsOption) ([]PinStatusGette
 	return res, <-errCh
 }
 
+// Manual version of Ls that returns a single batch of results and int with total count
+func (c *Client) LsBatchSync(ctx context.Context, opts ...LsOption) ([]PinStatusGetter, int, error) {
```

Comment:
Out of curiosity why are we returning `[]PinStatusGetter` here, is it actually of use given that it's just a single, non-resumable, page?